src/Marker.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  Marker.swift
+//  ios-sdk
+//
+//  Created by Sarah Lensing on 3/22/17.
+//  Copyright © 2017 Mapzen. All rights reserved.
+//
+
+import Foundation
+import TangramMap
+
+public enum MarkerType : Int {
+  case currentLocation, searchPin, routeLine
+}
+
+@objc(MZMarker)
+public class Marker: NSObject {
+
+  private static let kPointStyle = "points"
+  private static let kLineStyle = "lines"
+  private static let kPolygonStyle = "polygons"
+  private static let kDefaultBackgroundColor = UIColor.white
+  private static let kDefaultInteractive = true
+  private static let kDefaultSize = CGSize.zero
```

Comment:
This is how most UI components work in iOS (`UIButton`, `UIImage`). Unless you use the constructor which takes a size or image, it is zero. I think its okay to keep this as is because developers are used to other components behaving this way.